tests/ClientBuilderTest.php

```diff
@@ -161,12 +161,13 @@ public function testGetClient(): void
      */
     public function testIntegrationsAreAddedToClientCorrectly(bool $defaultIntegrations, array $integrations, array $expectedIntegrations): void
     {
-        $clientBuilder = new ClientBuilder([
```

Comment:
I would keep using the `array` of options so that we don't have to instantiate the `Options` object ourselves